modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -221,9 +222,14 @@ const makeRtcAdaptor = (rtcEditor: RtcRuntimeApi): RtcAdaptor => {
       formatChanged: (_rfl, formats, callback, similar, vars) => formatter.formatChanged(formats, callback, similar, vars)
     },
     editor: {
-      getContent: (args, _format) => editor.getContent(args),
+      getContent: (args) => editor.getContent(args),
       setContent: (content, args) => editor.setContent(content, args),
-      insertContent: (content, _details) => editor.insertContent(content),
+      insertContent: (content, details) => {
+        return preProcessSetContent(tinymceEditor, { content, format: 'html', set: false, selection: true, paste: details.paste }).each((args) => {
+          editor.insertContent(content);
+          postProcessSetContent(tinymceEditor, content, args);
+        });
+      },
```

Comment:
I like the changes you've done with set and get content, please do the same with insert content

- [ ] move insert content processing out of `Rtc.ts`